src/components/theme/View/View.jsx

```diff
@@ -202,7 +202,9 @@ export default class View extends Component {
       <div id="view">
         <Helmet
           bodyAttributes={{
-            class: `view-${RenderedView.displayName.toLowerCase()}`,
+            class: RenderedView.displayName
+              ? `view-${RenderedView.displayName.toLowerCase()}`
+              : 'view-undefined',
```

Comment:
@sneridagh, in which cases it gets undefined? Doesn't the new plugin always work?